packages/react-instantsearch-hooks/src/InstantSearchSSRProvider.tsx

```diff
@@ -22,6 +22,12 @@ export function InstantSearchSSRProvider({
   children,
   ...props
 }: InstantSearchSSRProviderProps) {
+  // If there's no server state passed, we don't want to override something
```

Comment:
What does "something" mean? Can we be as explicit as we can with our comments?

If I understand the code it's because we want to override the `<InstantSearchSSRProvider>` context value in the refetching of `getServerState()`: https://github.com/algolia/react-instantsearch/blob/785954e1eeab1eb1d21339ce5a6d9c2a3beab622/packages/react-instantsearch-hooks-server/src/getServerState.tsx#L88-L95